libmamba/src/core/repo.cpp

```diff
@@ -16,145 +21,136 @@ extern "C"  // Incomplete header
 
 #include "mamba/core/channel.hpp"
 #include "mamba/core/context.hpp"
+#include "mamba/core/mamba_fs.hpp"
 #include "mamba/core/output.hpp"
 #include "mamba/core/package_info.hpp"
 #include "mamba/core/pool.hpp"
 #include "mamba/core/prefix_data.hpp"
 #include "mamba/core/repo.hpp"
-#include "mamba/core/util_string.hpp"
+#include "solv-cpp/pool.hpp"
+#include "solv-cpp/repo.hpp"
 
 
-#define MAMBA_TOOL_VERSION "1.1"
+#define MAMBA_TOOL_VERSION "1.3"
```

Comment:
What is this version? libmamba is version 1.4.x at this point

Reply:
This is the version of the binary serialization of libsolv data. We slap our own version into it to update when we changes the internal libsolv data.

Probably not the best way to do this, because you need to deserialize it to be able to read it. We should think of something better...